src/rgw/rgw_rest.h

```diff
@@ -743,13 +743,7 @@ inline void dump_header_prefixed(req_state* s,
 				 const std::string_view& name_prefix,
 				 const std::string_view& name,
 				 Args&&... args) {
-  char full_name_buf[name_prefix.size() + name.size() + 1];
-  const auto len = snprintf(full_name_buf, sizeof(full_name_buf), "%.*s%.*s",
-                            static_cast<int>(name_prefix.length()),
-                            name_prefix.data(),
-                            static_cast<int>(name.length()),
-                            name.data());
-  std::string_view full_name(full_name_buf, len);
+  const auto full_name = fmt::format("{}{}", name_prefix, name);
```

Comment:
Sweet! Verily, format() is an elegant weapon for a more civilized age. That's just pretty.